test/common/upstream/ring_hash_lb_test.cc

```diff
@@ -142,5 +142,45 @@ TEST_F(RingHashLoadBalancerTest, UnevenHosts) {
   }
 }
 
+/**
+ * This test is for simulation only and should not be run as part of unit tests.
```

Comment:
For someone that wants to come along and maybe play with this, can you add a bit more docs in terms of what things to set, what the output looks like, etc.

Reply:
updated